tests/integration/python/test_python.py

```diff
@@ -0,0 +1,36 @@
+from tests.integration.base import DBTIntegrationTest, use_profile
+import os
+import pytest
+
+
+@pytest.mark.skip(
+    reason="Run manually. Test must start with the Python compute\
+          resource in TERMINATED or TERMINATING state"
+)
+class TestPython(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "python"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def project_config(self):
+        return {
+            "config-version": 2,
+            "vars": {"http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH")},
+        }
+
+    def python_exc(self):
+        self.run_dbt(["run"])
+        self.run_dbt(["run"])  # make sure it also run in incremental
```

Comment:
for the purpose of this test you can remove the second run

Reply:
Done in 4c767f4dd6be4f83b4974df6c7366fbf843d4572